index.html

```diff
@@ -1,9 +1,43 @@
-Ljiljana Janjic
-About
-I am a student of Code the Dream, currently enrolled in “Intro to programming” course. Based in Boston, MA (US), but originally from Sarajevo, Bosnia and Herzegovina. Prior Education: Master in Business Communication, Bachelor in Journalism and High-school degree in Computer science.
-Experience
-I do not have prior coding experience but insterested in computers, team player, problem-solving oriented (especially solving through working code), and learning new trends in programming as student of Code the Dream.
- Connect
- ljiljajanjic@gmail.com, https://github.com/Ljanjic, https://www.linkedin.com/in/ljiljana-lilla-janjic/
- /Users/ljiljanajanjic/homework/eridanus-intro/index.html
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>www.ljiljanajanjic.com</title>
+    </head>
+  <body>
+<section>
 
+    <img src="images/ljiljana-janjic-photo.jpg" width = '250'>
+    <h1><strong><em>Hello! I'm Ljiljana! Welcome to my Web page!</em></strong></h1>
+    
+    <p>I am a student of <a href="https://codethedream.org/"><strong>Code the Dream</strong></a>, currently enrolled in <strong>“Intro to programming”</strong> course. 
+        Based in <em>Boston, MA (US)</em>, but originally from <a href="https://www.visitsarajevo.ba/"><em>Sarajevo, Bosnia and Herzegovina</em></a>.</p>
+</section>
+
+<section>
+<h1><strong>Prior Education</strong></h1>
+<ul>
+    <li><em>Master in Business Communication</em></li>
+    <li><em>Bachelor in Journalism</em></li>
+    <li><em>High-school degree in Computer science</em></li>
+</ul>
+</section>
+
+<section>
+    <h1><strong>Experience</strong></h1>
+    <p>Through High-school education I learned basics of Programming by exploring <em>C++</em> language. 
+        Although I do not have prior (professional) experience I can describe myself as highly motivated and interested to: learn, develop new skills and knowledge, 
+        good <em>team player</em> and <em>problem-solving oriented</em> woman in Tech.</p>
+</section>
+
+<section>
+    <h1><strong>Let's Connect!</strong></h1>
+    <ul>
+       <h3><li><a href="www.linkedin.com/in/ljiljana-lilla-janjic"><strong>LinkedIn</strong></a></li></h3>
```

Comment:
This link won't open linkedin. Add https:// at the beginning of the URL and it will